### PR TITLE
Private authorization

### PR DESCRIPTION
**Pull request by Monicalyy**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Search/Search.sln [hunk 1]
```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26228.4
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Management.Search", "Management\Management.Search\Management.Search.csproj", "{A52AF58B-7383-416B-9A7E-601B9C628CDA}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Management.Search", "Management\Management.Search\Microsoft.Azure.Management.Search.csproj", "{A52AF58B-7383-416B-9A7E-601B9C628CDA}"
```
Comment (darshanhs90) [inline src/SDKs/Search/Search.sln]: >h\Microsoft.Azure. [](start = 101, length = 18)

why was this file modified?

File: src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Management.Search</PackageId>
+    <PackageId>Microsoft.Azure.Management.Search</PackageId>
     <Description>Makes it easy to manage Azure Search services and API keys from a .NET application.</Description>
     <AssemblyTitle>Microsoft Azure Search Management Library</AssemblyTitle>
-    <AssemblyName>Management.Search</AssemblyName>
+    <AssemblyName>Microsoft.Azure.Management.Search</AssemblyName>
```
Comment (darshanhs90) [inline src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj]: >Microsoft.Azure.Management.Search< [](start = 18, length = 34)

why was the search SDK modified?

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.5.0-preview" />
-    <ProjectReference Include="..\Management.Search\Management.Search.csproj" />
+    <ProjectReference Include="..\Management.Search\Microsoft.Azure.Management.Search.csproj" />
```
Comment (darshanhs90) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: >\Microsoft.Azure.Management.S [](start = 51, length = 29)

why was the search SDK modified?

**Comment by darshanhs90** [comment]
can you run the test cases also and include the new session records

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (Monicalyy) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: This is not my change, I just merged for vs17Dev.

File: src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj (hunk 2 above)
Comment (Monicalyy) [inline src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj]: This is not my change, I just merged for vs17Dev.

File: src/SDKs/Search/Search.sln (hunk 1 above)
Comment (Monicalyy) [inline src/SDKs/Search/Search.sln]: This is not my change, I just merged for vs17Dev.

Comment (darshanhs90) [inline src/SDKs/Search/Search.sln]: Can you checkout this Search SDK files from the upstream,just to be sure

Comment (Monicalyy) [inline src/SDKs/Search/Search.sln]: How can I check out the change? I just merged code form vs17Dev. it should not be include my changes, But GitHub still show these changes. Maybe if I need to create a new pull request?

Comment (andycnzh) [inline src/SDKs/Search/Search.sln]: It seems like GitHub pull request present issue. I don't see those changes locally.

Execute "git diff vs17Dev..Private-Authorization > diff.txt", and open this file using VS, don't see search SDK files.

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (markcowl) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: @Monicalyy Please checkout this file form psSdkJson6 and update the PR

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (Monicalyy) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: Ok. Will do it.

**Comment by shahabhijeet** [comment]
@Monicalyy this PR will be closed by EOW.
If closed, please reopen the same PR once you get a chance to fix the issue and feedback you got so far on this PR.

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (Monicalyy) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: @msftclas  I have revert the changes which merged from vs17Dev.

**Comment by Monicalyy** [comment]
@markcowl @shahabhijeet I have revert the changes which merged from vs17Dev.

**Comment by shahabhijeet** [comment]
@Monicalyy please reopen once you have address the PR.
Closing as this is very old PR
